index.js

```diff
@@ -127,25 +144,27 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
-
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    let userInventory = message.author.id;
+    let inventoryRef = "inventory";
+    let inventoryNames;
+    let userInventoryAccess = userData[userInventory][inventoryRef];
```

Comment:
this feels a bit unnecessary, we can just do `userData[userInventory].inventory`. By doing that we can get rid of the `inventoryRef` variable


In general you want to reduce the amount of lines of code you push, except when adding lines will add clarity.